water-box/water-box.ipynb

```diff
@@ -109,8 +118,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plt.plot(trj.time, rho, 'b-')\n",
-    "plt.plot(t_ref, rho_ref, 'k-')"
+    "fig, ax = plt.subplots()\n",
+    "\n",
+    "ax.plot(trj.time, rho, 'b-', label='Data')\n",
+    "ax.plot(ref[:, 0], ref[:, 1], 'k-', label='Reference')\n",
+    "\n",
+    "ax.set_xlabel(('Simulation time, ps'))\n",
+    "ax.set_ylabel(('System density, kg/m^3'))\n",
```

Comment:
The density seems to have periodic oscillations. Is this from Parrinello-Rahman + unequilibrated starting configuration? It may not matter for the purposes of this demo, just an observation.